lib/jekyll/commands/new.rb

```diff
@@ -114,6 +145,11 @@ def site_template
         def scaffold_path
           "_posts/0000-00-00-welcome-to-jekyll.markdown.erb"
         end
+
+        def user_email
```

Comment:
In this PR, I'm assigning user_email to an instance variable to be output by the ERB file.
In theme_builder, user_email is defined as a symbol which is forwarded to another subclass ERBRenderer. Since that symbol needn't be in the memory at the instant `jekyll new` is called, i decided to add a separate private method to this command.